app/common/expressions/managed.py

```diff
@@ -15,7 +15,7 @@
 
 
 class ManagedExpression(BaseModel):
-    key: ManagedExpressionsEnum
+    _key: ManagedExpressionsEnum
```

Comment:
I think we can tag this as a ClassVar to hide it from (de)serialize, see 6a334e36